scene/realistic-lighting-and-shadows/src/main/resources/realistic_lighting_and_shadows/style.css

```diff
@@ -17,10 +17,6 @@ VBox {
 
 .slider {
   -fx-padding: 10;
```

Comment:
we're still working out the sample viewer details: I'm not sure if leaving the padding in here will work ok in there. To be on the safe side, let's also move that into FXML (@JonLavi unless you have experience of the padding here having worked before?). Sorry for the uncertainty around this for now: still a work in progress for us to figure out!

Reply:
I haven't noticed this in the SV before, but to be fair it's all new territory, as you said. The easy way would be to leave it in the FXML, yes